pkg/objstore/inmem/inmem.go

```diff
@@ -57,23 +84,31 @@ func (b *Bucket) Iter(_ context.Context, dir string, f func(string) error) error
 
 // Get returns a reader for the given object name.
 func (b *Bucket) Get(_ context.Context, name string) (io.ReadCloser, error) {
+	if name == "" {
+		return nil, errors.New("inmem: object name is empty")
+	}
+
 	file, ok := b.objects[name]
 	if !ok {
-		return nil, errors.Errorf("no such file %s", name)
+		return nil, errNotFound
 	}
 
 	return ioutil.NopCloser(bytes.NewReader(file)), nil
 }
 
 // GetRange returns a new range reader for the given object name and range.
 func (b *Bucket) GetRange(_ context.Context, name string, off, length int64) (io.ReadCloser, error) {
+	if name == "" {
+		return nil, errors.New("inmem: object name is empty")
```

Comment:
can you add name to this error wessage?

Reply:
well, it is empty so not sure what should I add xD